theseus_gui/src/pages/instance/Index.vue

```diff
@@ -197,6 +201,15 @@ const checkProcess = async () => {
   uuid.value = null
 }
 
+// Get information on associated modrinth versions, if any
+const modrinthVersions = ref([])
+if (instance.value.metadata.linked_data) {
+  modrinthVersions.value = await useFetch(
+    `https://api.modrinth.com/v2/project/${instance.value.metadata.linked_data.project_id}/version`,
+    'project'
+  )
+}
```

Comment:
does this work in offline mode?

Reply:
throws an error but should work. I added a check so that it simply keeps it empty if offline